src/main/java/teammates/common/datatransfer/questions/FeedbackContributionResponseDetails.java

```diff
@@ -202,14 +202,13 @@ private String getContributionQuestionResponseAnswerCsv(
 
     public Map<String, TeamEvalResult> getContribQnTeamEvalResult(FeedbackQuestionAttributes question,
             FeedbackSessionResultsBundle feedbackSessionResultsBundle) {
-        Map<String, TeamEvalResult> contribQnStats =
-                feedbackSessionResultsBundle.contributionQuestionTeamEvalResults.get(question.getId());
-        if (contribQnStats == null) {
+        feedbackSessionResultsBundle.contributionQuestionTeamEvalResults.computeIfAbsent(question.getId(), key -> {
+            Map<String, TeamEvalResult> contribQnStats =
+                    feedbackSessionResultsBundle.contributionQuestionTeamEvalResults.get(question.getId());
```

Comment:
okie nice catch here but if this line of code is executed, it means this `contribQnStats` is null and theres no point getting it.